### PR TITLE
Vue3  example

### PR DESCRIPTION
**Pull request by manang**
### Description
I've created an example of gridstack used in a vue3 js.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@
 .npmrc
 coverage
 dist
-node_modules
\ No newline at end of file
+node_modules
+.vscode
```
Comment (adumesny) [inline .gitignore:8]: please revert that

**Review by adumesny — CHANGES_REQUESTED** [review]
Angelo, thank you for doing this, but can you make it EXACTLY the same as the 2.0 version that is checked in (down to the style and includes of demo styles) other than `import { createApp }` and `.mount("#app")` part so it's easier to diff the two and see what changed for 3.0 ?

Also, please append your file to index.html and readme link, maybe rename the existing one vue2ls.html

and finally, merge all your git changes into just 1 if possible (git rebase -i HEAD~3 and squash, the git push -f to update your commits), otherwise I can do it when I accept I suppose.

File: .gitignore (hunk 1 above)
Comment (manang) [reply .gitignore:8]: why you want .vscode inside the repo?

Comment (adumesny) [reply .gitignore:8]: yeah you're right. never mind

**Pull request merged**